include/seqan3/range/container/concatenated_sequences.hpp

```diff
@@ -923,9 +926,18 @@ class concatenated_sequences
             value_len = std::distance(ranges::begin(value), ranges::end(value));
 
         data_values.reserve(data_values.size() + count * value_len);
+        auto placeholder = ranges::view::repeat_n(value_type_t<rng_type>{}, count * value_len)
+                         | view::common;
```

Comment:
The commit message is `[FIX] ensure that ranges passed on are actually ForwardRanges `, but I can't find where this is actually required in your first commit.

Reply:
this calls the same functions on the underlying range and they require forward